cylc/flow/scripts/lint.py

```diff
@@ -386,7 +389,7 @@ def get_upgrader_info():
 }
 
 
-def parse_checks(check_args, ignores=None, max_line_len=None):
+def parse_checks(check_args, ignores=None, max_line_len=130, reference=False):
     """Collapse metadata in checks dicts.
 
     Args:
```

Comment:
I'm not sure what "Collapse metadata in checks dicts" means.

The `reference` arg is missing a docstring entry

Reply:
Is `130` the default `max_line_len`?

Reply:
Yes, is that a sensible number - I wanted to make it a long but not insane length. I could instead make it v.v. big.